ad/resource_ad_user.go

```
@@ -291,11 +340,19 @@ func resourceADUserRead(d *schema.ResourceData, meta interface{}) error {
 	_ = d.Set("smart_card_logon_required", u.SmartcardLogonRequired)
 	_ = d.Set("trusted_for_delegation", u.TrustedForDelegation)
 
+	if u.CustomAttributes != nil {
+		ca, err := structure.FlattenJsonToString(u.CustomAttributes)
+		if err != nil {
+			return err
+		}
+		_ = d.Set("custom_attributes", ca)
+	}
+
 	return nil
 }
 
 func resourceADUserUpdate(d *schema.ResourceData, meta interface{}) error {
-	u := winrmhelper.GetUserFromResource(d)
+	u, _ := winrmhelper.GetUserFromResource(d)
```

Is it safe to ignore the error value here?